scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -74,22 +74,34 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             $scope.proofread = false;
             $scope.referrerUrl = referrer.getReferrerUrl();
 
-            if ($scope.origItem.task && $scope.origItem.task.stage) {
-                if (archiveService.isLegal($scope.origItem)) {
-                    $scope.deskName = $scope.origItem.task.desk;
-                    $scope.stage = $scope.origItem.task.stage;
-                } else {
-                    api('stages').getById($scope.origItem.task.stage)
-                        .then(function(result) {
-                            $scope.stage = result;
-                        });
+            /**
+             * Get the Desk and Stage for the item.
+             */
+            function getDeskStage() {
+                if ($scope.origItem.task && $scope.origItem.task.stage) {
+                    if (archiveService.isLegal($scope.origItem)) {
+                        $scope.deskName = $scope.origItem.task.desk;
+                        $scope.stage = $scope.origItem.task.stage;
+                    } else {
+                        api('stages').getById($scope.origItem.task.stage)
+                            .then(function(result) {
+                                $scope.stage = result;
+                            });
 
-                    desks.fetchDeskById($scope.origItem.task.desk).then(function (desk) {
-                        $scope.deskName = desk.name;
-                    });
+                        desks.fetchDeskById($scope.origItem.task.desk).then(function (desk) {
```

Comment:
You forgot this one. I will change all other files soon and add this to the linters.